src/Microsoft.Extensions.DependencyModel/CompilationOptions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Extensions.DependencyModel
+{
+    public class CompilationOptions
```

Comment:
Other model is mutable and has more properties that doesn't make sense (preserver compilation context)